### PR TITLE
Config test suite

### PR DESCRIPTION
**Pull request by KarenVentura**
- Config Rspec for unit test
- Config Cucumber for integration test  
- As well install and set up Spring  for bundle exec spring [cucumber or rspec]

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,9 +21,14 @@ gem 'uglifier', '>= 1.3.0'
...
   gem 'pry-nav'
+  gem 'rspec-rails', '~> 3.0'
+  gem 'cucumber-rails', :require => false
+  gem 'database_cleaner'
 end
 
 group :development do
   gem 'web-console', '~> 2.0'
-  gem 'spring'
+  gem 'spring', '~> 1.4.0'
```
Comment (mumoc) [inline Gemfile]: why using the version? is there any special issue?

File: features/support/env.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
```
Comment (mumoc) [inline features/support/env.rb]: Go ahead and remove all the comments, we won't need them xD

**Comment by mumoc** [comment]
The configuration looks good to me (LGTM), but before merging, check why is not  working the travis check ^^

File: Gemfile (hunk 1 above)
Comment (KarenVentura) [inline Gemfile]: I read that most stable version is that, but i just did a test and it is ok with the newest versions.

**Comment by sandrely26** [comment]
This PR needs travis.yml that's why it isn't working properly.
Also travis log said this:
/home/travis/.rvm/rubies/ruby-1.9.3-p551/bin/ruby -S bundle exec cucumber  --profile default
FATAL:  database "ratefast_test" does not exist

I think is because travis config.

**Pull request closed without merging**